sensors/bms/launch/bms_launch.py

```diff
@@ -0,0 +1,10 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+def generate_launch_description():
+    return LaunchDescription([
+        Node(
+            package='bms',
+            executable='bms_publisher'
```

Comment:
Read what I comented in setup.py file, this should be changed to freya_bms_node or freya_bms_publisher_node :)

Reply:
:+1: